src/pages/ErrorPage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Container, Image, VStack, Center } from '@chakra-ui/react';
+import './ErrorPage.css';
+import image2 from '../assets/images/image2.png';
+
+const ErrorPage = () => {
+  return (
+    <Container id="window">
```

Comment:
let's use `className`s instead of `id`s where applicable; it's generally better practice + works better with `module.css`